gem/templates/springster/surveys/molo_survey_page.html

```diff
@@ -179,7 +175,7 @@ <h6 class="surveys__pagination-title">
           <div class="{% if self.display_survey_directly %}polls{% else %}surveys{% endif %}__pagination">
               <p class="{% if self.display_survey_directly %}polls{% else %}surveys{% endif %}__page">
                 {% with page=fields_step.number pages=fields_step.paginator.num_pages %}
-                  {% trans Page %}<span class="{% if self.display_survey_directly %}polls{% else %}surveys{% endif %}__page-number">{{page}}</span> {% trans of %}<span class="{% if self.display_survey_directly %}polls{% else %}surveys{% endif %}__page-number">{{pages}}</span>
+                  {% trans "Page"  %}<span class="{% if self.display_survey_directly %}polls{% else %}surveys{% endif %}__page-number"> {{page}}</span> {% trans "of" %}<span class="{% if self.display_survey_directly %}polls{% else %}surveys{% endif %}__page-number"> {{pages}}</span>
```

Comment:
@sewagodimo not possible to use blocktrans here, the issue you raised per translated words order will not be fixed.

Reply:
I think the issue is the html tags in between the words